README.rdoc

```diff
@@ -67,6 +67,28 @@ To install:
 
     bundle exec rake install
 
+=== JRuby
+
+First, you need to download Apache Commons Math 3.6.1 (the JAR, which
+you can find in the binary package). For example, in the NMatrix
+directory, do:
+
+    wget https://www.apache.org/dist/commons/math/binaries/commons-math3-3.6.1-bin.tar.gz
+    tar zxvf commons-math3-3.6.1-bin.tar.gz
+    mkdir ext/nmatrix_java/vendor/
+    cp commons-math3-3.6.1/commons-math3-3.6.1.jar ext/nmatrix_java/vendor/
+
```

Comment:
Are all these steps still required? It seems like this is already done by the rake task.

Also is this the standard way of dealing with Java libraries (just downloading them and sticking the jar in a random directory)? It seems a little unfortunate.

Are you planning to get this to work with a simple `gem install`?

Also, you should probably add a small note to the README, noting the limited features of the JRuby version.